gcs/conversions.go

```diff
@@ -124,26 +124,29 @@ func toObject(in *storagev1.Object) (out *Object, err error) {
 	}
 
 	// CRC32C
-	crc32cString, err := base64.StdEncoding.DecodeString(in.Crc32c)
-	if err != nil {
-		err = fmt.Errorf("Decoding Crc32c field: %v", err)
-		return
-	}
-
-	if len(crc32cString) != 4 {
-		err = fmt.Errorf(
-			"Wrong length for decoded Crc32c field: %d",
-			len(crc32cString))
+	// No CRC32C or MD5 values returned for CMEK buckets
```

Comment:
Do you have a reference for this in the public docs?

Reply:
https://cloud.google.com/storage/docs/gsutil/addlhelp/UsingEncryptionKeys

IIUC - CRC32C and MD5 are available, but require a separate roundtrip to get / decrypt the metadata.  They're not returned as part of the standard request.  Adding the extra roundtrip would seem to have big performance overhead.

Reply:
Thanks. Could you change the comment to say something like:

    // Populate CRC32C when present in the response. This may be missing e.g. for
    // buckets using customer-managed encryption keys.
    // See: https://cloud.google.com/storage/docs/gsutil/addlhelp/UsingEncryptionKeys